demisto_sdk/commands/common/errors.py

```diff
@@ -4345,3 +4371,43 @@ def nativeimage_exist_in_script_yml(script_id):
             f"script {script_id} contains the nativeimage key in its yml, "
             f"this key is added only during the upload flow, please remove it."
         )
+
+    @staticmethod
+    @error_code_decorator
+    def uses_items_not_in_marketplaces(
+        content_name: str, marketplaces: list, content_items: list
+    ):
+        return (
+            f"Content item '{content_name}' with marketplaces: '{', '.join(marketplaces)}' is using content items: "
+            f"'{', '.join(content_items)}' that have incompatible marketplaces."
+        )
+
+    @staticmethod
+    @error_code_decorator
+    def uses_items_with_invalid_fromversions(
+        content_name: str, fromversion: str, content_items: list
+    ):
+        return (
+            f"Content item '{content_name}' with from_version: '{fromversion}' is using content items: "
+            f"'{', '.join(content_items)}' that have incompatible from_version."
```

Comment:
Conetnt item .. whose from_version is .. uses the content items ... , whose from_version is higher (must be equal to, or less than ..)